src/line.js

```diff
@@ -34,12 +36,17 @@ SVG.Line = SVG.invent({
       return this.attr(this.array().size(p.width, p.height).toLine())
     }
   }
-  
+
   // Add parent method
 , construct: {
     // Create a line element
     line: function(x1, y1, x2, y2) {
-      return this.put(new SVG.Line).plot(x1, y1, x2, y2)
+      var line = this.put(new SVG.Line)
+
+      // make sure plot is called as a setter
+      return (x1 != null) ?
+        line.plot(x1, y1, x2, y2) :
+        line.plot([[0,0], [0,0]])
```

Comment:
I like what you did with the boolean OR, I need to use that operator more often.